addon.json

```diff
@@ -1,14 +0,0 @@
-{
-  "addon": {
-    "version": "2.0.0",
-    "id": "online-news@pioneer.mozilla.org",
-    "name": "Pioneer Online News Study",
-    "minVersion": "55.0",
-    "maxVersion": "*",
-    "multiprocessCompatible": true,
-    "hasEmbeddedWebExtension": false,
-    "creator": "Mozilla",
-    "description": "A Pioneer study looking at online news consumption.",
-    "chrome": "pioneer-study-online-news-2"
-  }
-}
```

Comment:
Deleted this file and moved the config into package.json (to help reduce some duplicated strings and settings)